build.yaml

```diff
@@ -16,9 +16,8 @@ targets:
           dart2js_args:
           - --minify
           - --fast-startup
-          - -DPRE_NULL_SAFETY_SERVER_URL=https://v1.api.dartpad.dev/
-          - -DNULL_SAFETY_SERVER_URL=https://nullsafety.api.dartpad.dev/
-          - -DSTABLE_SERVER_URL=https://v1.api.dartpad.dev/
+          - -DSERVER_URL=https://nullsafety.api.dartpad.dev/
+          - -DSTABLE_SERVER_URL=https://nullsafety.api.dartpad.dev/
```

Comment:
@srawlins @domesticmouse are we going to change this to `stable.api.dartpad.dev`?

Reply:
my plan is to shutdown `nullsafety.api.dartpad.dev`. Stable and v1 are the same thing, and will be serving null safety.

Reply:
Maybe we should change this from `nullsafety` to `stable`?

Reply:
done